packages/internal/src/routes.ts

```diff
@@ -0,0 +1,97 @@
+import fs from 'fs'
+
+import chalk from 'chalk'
+
+import { getPaths } from './paths'
+
+export interface Route {
+  name?: string
+  path?: string
+  page?: string // All valid routes should have a page but it's optional here for convenience
+}
+
+/**
+ * Extracts all the routes defined within the Routes.(js|tsx) file
+ */
+export function getRoutes() {
```

Comment:
Instead of this, I wonder if we could use `packages/structure/src/model/RWRouter.ts` What do you think?

Reply:
I can change to use this certainly. I assume you prefer I did rather than have an, I guess, unnecessary additional routes extraction logic?

Reply:
Correct. I also think it's more robust in that it doesn't rely on regexes but rather uses an abstract syntax tree (AST)